src/test/java/uk/gov/companieshouse/extensions/api/requests/MongoDBTest.java

```diff
@@ -77,7 +77,7 @@ public void removeTestDocuments() {
         requestsRepository.deleteById(SUMMER_TEST_DOCUMENT);
     }
 
-    @Ignore
+    @Ignore("This requires an in-memory db similar to Derby but for Mongo")
```

Comment:
looks like these are ignored as we need a mongoDB connection/db to run these and concourse doesn't have one. I was thinking, as this test is marked with a category of "integration", concourse will try and execute it. Perhaps for a future Jira story, we could look into having a separate category for this and other DB tests that we could add into the makefile for running locally only as we'll forget that we have to un-ignore them.